src/test/java/io/localmotion/chatbox/notification/ChatBoxNotificationRepositoryTest.java

```diff
@@ -0,0 +1,176 @@
+package io.localmotion.chatbox.notification;
+
+import io.micronaut.test.annotation.MicronautTest;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.transaction.PlatformTransactionManager;
```

Comment:
why are use using spring here?

Reply:
first of all, Micronaut is rely on Spring Transaction Management. I need to clean the In memory database for each test case in an entirely different transaction. 

Since I am testing my Repository, It was needed to start a new transaction.

suggestions always welcome.